pkg/services/ngalert/schedule/schedule.go

```diff
@@ -83,27 +77,20 @@ type schedule struct {
 
 	evaluator eval.Evaluator
 
-	ruleStore        store.RuleStore
-	instanceStore    store.InstanceStore
-	adminConfigStore store.AdminConfigurationStore
-	orgStore         store.OrgStore
+	ruleStore     store.RuleStore
+	instanceStore store.InstanceStore
+	orgStore      store.OrgStore
 
 	stateManager *state.Manager
 
 	appURL               *url.URL
 	disableGrafanaFolder bool
 
-	multiOrgNotifier *notifier.MultiOrgAlertmanager
-	metrics          *metrics.Scheduler
+	metrics *metrics.Scheduler
 
-	// Senders help us send alerts to external Alertmanagers.
-	adminConfigMtx          sync.RWMutex
-	sendAlertsTo            map[int64]ngmodels.AlertmanagersChoice
-	sendersCfgHash          map[int64]string
-	senders                 map[int64]*sender.Sender
-	adminConfigPollInterval time.Duration
-	disabledOrgs            map[int64]struct{}
-	minRuleInterval         time.Duration
+	alertsSender    AlertsSender
```

Comment:
My last comment that echoes what I've said as part of https://github.com/grafana/grafana/pull/48144#discussion_r918245869 (and still remains unaddressed) the comment there seems to be orthogonal is that I still don't agree with the name `alertsSender` here.

This is the router and not the sender. The sender is an internal component that the router uses and therefore it's confusing.

A suggestion is to change it to router - you don't need the alerts prefix because it can be `sender.Router` and the interface itself can come from the other package if it helps simplify things.

If you don't like my suggestion then I'd hope to see it changed to something else that's not `alertsSender` for both the attribute and the interface because *we already have a sender* that's being used within the router.

Reply:
Discussed this offline with Yuri, it seems like the proposal here is that we'll change the name of the sender within the router to something different in the next PR - I'm OK with that as long as it doesn't remain in this state for too long.